scripts/scil_compare_connectivity.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Performs a network-based statistical comparison for populations G1 and G2 using
+a t-statistic threshold of alpha. All matrices must have the same shape.
+
+For example, if you have streamline count weighted matrices for a MCI and a
+control group and you want to investiguate difference in their connectomme:
+    scil_compare_connectivity.py --g1 MCI/*_sc.npy --g2 CTL/*_sc.npy
```

Comment:
Your example is "wrong". It should be --in_g1 and --in_g2. Otherwise, change your options below.